web/src/hotkey.js

```diff
@@ -241,10 +241,10 @@ export function process_escape_key(e) {
     if (
         recent_topics_ui.is_in_focus() &&
         // This will return false if `e.target` is not
-        // any of the recent topics elements by design.
+        // any of the Recent Conversations elements by design.
         recent_topics_ui.change_focused_element($(e.target), "escape")
     ) {
-        // Recent topics uses escape to switch focus from RT search / filters to topics table.
+        // Recent Conversations uses escape to switch focus from RT search / filters to topics table.
         // If focus is already on the table it returns false.
```

Comment:
This rest of this comment needs an update too! `RT`, for example, and "topics table".